greedy_approach/dijkstra.c

```diff
@@ -31,11 +33,15 @@ int queue_has_something() { return (qp > 0); }
 
 int visited[MAX];
 int vp = 0;
-
+// Function that implements Dijkstra's single source
+// shortest path algorithm for a graph represented using
+// adjacency matrix representation
 void dijkstra(int s)
```

Comment:
This is not the proper way of documenting functions. Check our [**typical structure of a program**](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#typical-structure-of-a-program).